vochain/apputils.go

```diff
@@ -62,6 +63,7 @@ func NewNodeKey(tmPrivKey, nodeKeyFilePath string) (*tmtypes.NodeKey, error) {
 	if err != nil {
 		return nodeKey, fmt.Errorf("cannot decode private key: (%s)", err)
 	}
+	// We need to use ed25519 curve for node key since tendermint does not support secp256k1
```

Comment:
Do we want to add a TODO in this part? It is somewhat for me strange that a validator specifies a `secp256k1` key but internally uses a `ed25519`. I feel that is a bit hacky and for consistency, the same crypto should be used.

Reply:
This is a limitation on the Tendermint side. Nothing to do.